style.css

```diff
@@ -316,6 +293,172 @@
     border: 1px solid #344563;
   }
 
+  /* Detail Section Starts Here */
+
+  .detail-card {
```

Comment:
- Kindly make your project matching with the Figma design, fix the size of the card 

- Kindly correct the position of the elements, as you can see on the Figma design, the text is on the top and bottom in the bottom


Your design

![image](https://github.com/sulaimonaa/Portfolio/assets/93630700/fe5c3c85-9f7a-474b-87c1-bd6cdc2f24c1)


Figma design

![image](https://github.com/sulaimonaa/Portfolio/assets/93630700/b75b717d-b43c-4672-b5f3-3ae6eb39d4fc)